StarkInfra/StarkInfra/CreditNote/Log.cs

```diff
@@ -18,15 +18,15 @@ public partial class CreditNote
         /// Properties:
```

Comment:
Na linha 14, na descrição do CreditNote.Log está faltando uma parte. 

Deveria estar assim: 
`Every time a CreditNote entity is updated, a corresponding creditnote.Log
is generated for the entity. This log is never generated by the
user, but it can be retrieved to check additional information
on the CreditNote.`